door_tracker/tracker.py

```diff
@@ -0,0 +1,94 @@
+import cv2
+import sys
+import time
+
+PATH = r'..\closed_open_vid.mp4'
+
+if __name__ == '__main__':
+
+    tracker_type = 'KCF'
+    tracker1, tracker2 = (cv2.TrackerKCF_create(), cv2.TrackerKCF_create())
+
+    video = cv2.VideoCapture(PATH)  # 0 instead of PATH for CAM
+
+    if not video.isOpened():
+        print("Could not open video")
+        sys.exit()
+
+    ok_frame, frame = video.read()
+    if not ok_frame:
+        print('Cannot read video file')
+        sys.exit()
+
+    # Define an initial bounding box (x, y, w, h)
+    length = 90
+    x = 619
+    y = 236
+    initial_bbox1 = (x + 430 - length, y - length//2, length, length)
```

Comment:
why do tou use fixed number?
souldnt you recive them as paramaters?

Reply:
fixed